xml/src/test/resources/configs/simple-service.xml

```diff
@@ -0,0 +1,45 @@
+<!--
+  ~ Copyright Terracotta, Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<config
+  xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
+  xmlns='http://www.ehcache.org/v3'
+  xmlns:ss="http://www.ehcache.org/v3/ss"
+  xsi:schemaLocation="http://www.ehcache.org/v3 ../../../../../xml/src/main/resources/ehcache-core.xsd
+                        http://www.ehcache.org/v3/ss ../../../../../xml/src/main/resources/ehcache-simple-service-ext.xsd">
+
+  <service>
+    <ss:instance class="org.ehcache.core.spi.time.TickingTimeSource">
+      <ss:arg value="10"/>
```

Comment:
This is nice... but it would be better if we could name the parameters in the XML and have them matched against the names of the constructor parameters. Then in cases where the names aren't available via reflection and we have to resort to ordering (and the types are indistinct) we could print a big warning?

Reply:
That sounds like a lot of fuss for little value IMHO.